filecoin-proofs/src/caches.rs

```diff
@@ -62,14 +62,6 @@ where
     Ok(res)
 }
 
-#[inline]
-pub fn lookup_groth_params<F>(identifier: String, generator: F) -> Result<Arc<Bls12GrothParams>>
-where
-    F: FnOnce() -> Result<Bls12GrothParams>,
-{
-    cache_lookup(&*GROTH_PARAM_MEMORY_CACHE, identifier, generator)
```

Comment:
Why did you remove the use of the cache? shouldn't we still use that?

Reply:
Compiler was telling me it wasn't used.

Reply:
I mean, you should keep using it

Reply:
Ok, let me take a look, thanks.